app/services/candidates/registrations/placement_request_action.rb

```diff
@@ -37,7 +37,13 @@ def application_preview
       end
 
       def school_contact_email
-        registration_session.school.contact_email
+        if registration_session.school.profile &&
```

Comment:
Thoughts on moving this conditional into the school so we can go `registration_session.school.email_for_notifications` ?